nifi-weaver-processors/src/test/java/com/weaverplatform/nifi/XmiImporterTest.java

```diff
@@ -32,8 +33,6 @@
 import java.io.InputStream;
 import java.util.List;
 
-import static org.junit.Assert.*;
-
```

Comment:
A Test should really have asserts, the fact that this import can be removed is troubling :/